### DISCUSSION TITLE
Sharing my review of Graph Analysis

### DISCUSSION THREAD

**Discussion by ostrichery** [SkepticMystic/graph-analysis#61] [Show and tell]
Hello! I love this plugin and tried a few comparisons to see how the algorithms differ from each other. Was really a blast to have to try to understand (as much as I could) what each was trying to do. Have a look! 
https://medium.com/@ensleytan/obsidians-graph-analysis-plugin-c9c107da3331?source=friends_link&sk=4eefee05f7929ba0b38965bc3d535235

**Comment by SkepticMystic** [comment]
Oh wow, very cool!
Thanks for sharing :)
